code/obj/item/misc_weapons.dm

```diff
@@ -495,18 +502,22 @@ TYPEINFO(/obj/item/sword/pink/angel)
 		. += "It is set to [src.active ? "on" : "off"]."
 
 /obj/item/sword/discount/attack(mob/target, mob/user, def_zone, is_special = FALSE, params = null)
-	//hhaaaaxxxxxxxx. overriding the disorient for my own effect
-	if (active)
-		hit_type = DAMAGE_BURN
-	else
-		hit_type = DAMAGE_BLUNT
+
+	if (royale_mode)
+		if (active)
+			hit_type = DAMAGE_BURN
+		else
+			hit_type = DAMAGE_BLUNT
```

Comment:
Gamemode switches like this are bad, you've already made a subtype so either put a var or proc override on that instead. Orrr maybe just change it globally since the changes probably also make sense in BR and having two identical versions of the same weapon that work differently is Bad.

Reply:
can do. just wanted to have all bases covered for not buffing it in BR. but its probably ok